package.json

```diff
@@ -19,14 +19,15 @@
   "scripts": {
     "old": "node build.js && rollup -c",
     "build": "node build_filterNames",
-    "wikimatch": "node match_wikiTags"
+    "wikimatch": "node match_wikiTags",
+    "test": "node validate"
   },
   "devDependencies": {
     "clear": "^0.1.0",
     "colors": "^1.1.2",
     "diacritics": "^1.3.0",
     "json-stringify-pretty-compact": "^1.1.0",
-    "jsonschema": "^1.1.0",
+    "jsonschema": "^1.2.4",
```

Comment:
Probably/Shouldn't break anything, but why the update?

Reply:
Why not? It is always good to use the latest version to avoid security issues...

Reply:
Fair point!